src/index.js

```diff
@@ -40,6 +40,10 @@ export default function commonjs(options = {}) {
 			} catch (err) {
 				resolvedId = resolve(id);
 			}
+
+			// Note: customNamedExport's keys must be normalized file paths.
+			// resolve and nodeResolveSync both return normalized file paths
+			// so no additional normalization is necessary.
```

Comment:
Yes, I'm cool with this, this is educational and will help people not familiar with the code 👍